x/feemarket/spec/02_state.md

```diff
@@ -0,0 +1,45 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Block Gas Used
+
+The total gas used by current block is stored in the KVStore at `EndBlock`.
+
+It is initialized to `block_gas` defined in the genesis.
+
+## Base Fee
+
+### NoBaseFee
+
+We introduce two parameters : `NoBaseFee`and `EnableHeight`
+
+In case `NoBaseFee = true` or `height < EnableHeight`, the base fee value will be equal to `base_fee` defined in the genesis.
+
+Those parameters allow us to introduce a static base fee or activate the base fee at a later stage.
+
+### Calculation
+
+The base fee is initialized at `EnableHeight` to the `InitialBaseFee` value defined in the genesis file.
+
+The base fee is after adjusted according to the total gas used in the previous block.
+
+```golang
```

Comment:
It'd be helpful to add a few comments to this calculation code snippet. For example, I assume `parent_*` refers to the previous block numbers, right? (If so, perhaps renaming it to `previous_block_...` could make it more explicit? )
The other thing worth explaining may be constants, such as `ELASTICITY_MULTIPLIER`, how they are set.

Reply:
ok, it seems they are not constants, but network parameters . Perhaps it's good to highlight that and explain in x/feemarket/spec/05_params.md